tests/test_query_engine_dsl.py

```diff
@@ -303,3 +303,91 @@ def test_date_arithmetic_conversions(engine, cohort_with_population):
         {"patient_id": 8, "age_in_years": 9, "age_in_months": 119},
         {"patient_id": 9, "age_in_years": 9, "age_in_months": 118},
     ]
+
+
+@pytest.mark.parametrize(
+    "current_date,age_data",
+    [
```

Comment:
I'd be interested to know more about why you chose these dates and, as a consequence, what cases we're testing. Am I correct in thinking that you chose the end dates to be after, before, and equal to February because February may have a leap day?

Reply:
I did choose dates around Feb because of Feb being fewer days, but the real reason is that the initial iteration of the julian days version of this treated Jan and Feb differently to the rest of the months (it used a formula that involved adding 12 to the month and subtracting 1 from the year if the month is <=2.  Because apparently the year should really start on 1st March, which makes sense really, because then October is month 8, November is month 9...)

THe comments about the end month are probably misleading now that there's nothing special about month 2, I'll take them out